libsentrykube/tests/test_data/quickpatches/test-patch.yaml.j2

```diff
@@ -0,0 +1,29 @@
+# Each patch file defines a patch for a single resource
```

Comment:
Should we consider adding a test which validates that the quickpatch required properties exists on all quickpatch files?